Pod/Classes/Renderer/PDFViewController.swift

```diff
@@ -91,6 +91,7 @@ open class PDFViewController: UIViewController {
         self.collectionView.collectionViewLayout.invalidateLayout()
         
         self.view.layoutSubviews()
+        collectionView.reloadItems(at: [IndexPath(row: 0, section: 0)])
```

Comment:
While this does solve the issue, it also looks like it is causing the page to blink (re-render) every time you scroll to it